### PR TITLE
Refresh the list of Gitpod examples

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

File: src/components/index/GetStarted.tsx [hunk 1]
```diff
@@ -33,22 +33,22 @@ const StyledGetStarted = styled.div`
...
+      flex-wrap: wrap;
 
       @media (max-width: ${sizes.breakpoints.lg}) {
         overflow-x: scroll;
+        justify-content: space-between;
+        flex-wrap: nowrap;
       }
 
       & > .project {
         margin-bottom: 3rem;
```
Comment (gtsiolis) [inline src/components/index/GetStarted.tsx:46]: **nitpick**: Go symmetry! 😅 

```suggestion
        margin-bottom: 5rem;
```

**Review by gtsiolis — COMMENTED** [review]
Aha! 💯

File: src/components/index/GetStarted.tsx (hunk 1 above)
Comment (jankeromnes) [reply src/components/index/GetStarted.tsx:46]: Agreed!

Gah, no one-click accept in Gitpod:

Screenshot 2020-12-08 at 17 31 12

Comment (jankeromnes) [reply src/components/index/GetStarted.tsx:46]: Wow, I just got super-confused by my own screenshot, and was about to report a Theia bug:

Screenshot 2020-12-08 at 17 33 31

**Comment by jankeromnes** [comment]
Works great! It seems all examples work great (I have a PR for NuShell) and have prebuilds enabled, so I'll merge this now. 🚀

**Pull request merged**